AnimatedWordCloud/Utils/Consts.py

```diff
@@ -23,3 +23,7 @@
     "Fonts",
     "NotoSansMono-VariableFont_wdth,wght.ttf",
 )
+
+TMP_OUTPUT_PATH = os.path.join(
```

Comment:
rename `TMP_OUTPUT_PATH` into not like TEMP

just make the actual directory name "tmp_output", but not the const name